src/components/ConnectedFilter/utils.js

```diff
@@ -42,9 +43,92 @@ const capitalizeFirstLetter = (str) => {
   return res.replace(/\w\S*/g, (txt) => txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase());
 };
 
+// createSearchFilterLoadOptionsFn creates a handler function that loads the search filter's
+// autosuggest options as the user types in the search filter.
+const createSearchFilterLoadOptionsFn = (field, guppyConfig) => (searchString, offset) => {
+  const NUM_SEARCH_OPTIONS = 20;
```

Comment:
should we make this field available in guppy config file?

Reply:
The configured field would need to go in portal's `guppyConfig` and then get passed to GuppyWrapper, right? I thought it was too much trouble to change the portal config for, especially if we're going to replace this feature with free text search down the line. I'm open to making it configurable if you feel differently though!

Reply:
oh i was think sth to just put in manifest like https://github.com/uc-cdis/guppy#configurations

but you are right about the point that it is maybe still subject to change in the future

Reply:
> oh i was think sth to just put in manifest like https://github.com/uc-cdis/guppy#configurations

Wouldn't that not work in this case because this is frontend code? I thought that the manifest.json config only applied to the backend guppy code, while the frontend guppy code gets pulled from npm when portal's rolled.

Reply:
well yeah i was think sth like `tier_access_limit`, but you are right, this should be a purely FE config if it exists 😅